app/views/collections/show.html.erb

```diff
@@ -11,12 +11,9 @@
           <%= sanitize @presenter.description %>
         </div>
       </header>
-      <% unless has_collection_search_parameters? %>
-      <%= render 'collections/show_descriptions' %>
-      <% end %>
     </div>
     <div class="col-sm-2">
-      <%= render partial: 'collections/media_display', locals: { generic_file: @collection } %> 
+      <%= image_tag url_for(:controller => 'communities', action: 'logo', id: @collection.id), alt: "No logo" %>
```

Comment:
It seems that there's no control over the size of the image allowed on the upload/display. I uploaded a medium size logo file for testing, and it overlapped with the community title. We will probably need both some css work to control the allowed size.